File: solutions/0392-is-subsequence/solution.py
class Solution(object):
    def isSubsequence(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        n=len(s)
        j=0
        for i in t:
            if j<n:
                if i==s[j]:
                    j+=1
        return True if j==n else False

File: solutions/0392-is-subsequence/solution.java
class Solution {
    public boolean isSubsequence(String s, String t) {
        char[] arr=s.toCharArray(),brr=t.toCharArray();
        int n=arr.length,j=0;
        for(char i:brr)
            if(j<n)
                if(i==arr[j])
                    j+=1;
        return (j==n)?true:false;
    }
}
